gk/main.c

```diff
@@ -2442,6 +2443,8 @@ gk_stage1(void *arg)
 		}
 		inst_ptr->tx_queue_back = ret;
 
+		instance->sol_inst = &sol_conf->instances[i % sol_conf->num_lcores];
```

Comment:
We need to do something smarter here (and/or at the Lua configuration) in order to match the NUMA node of the GK blocks and their corresponding SOL block.

I don't have a solution, but it's better to discuss it before implementing it.

Reply:
A solution would be for `struct gk_config` to have an array whose index is the instance and the value the index of the associated SOL instance. This solution pushes the association to the Lua code, so it can make the NUMA node match or any other match that we may want to try.